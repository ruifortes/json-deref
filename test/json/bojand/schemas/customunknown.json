{
  "description": "Just a basic schema.",
  "title": "Basic Object",
  "type": "object",
  "definitions": {
    "bar": {
      "description": "bar property",
      "type": "boolean"
    }
  },
  "properties": {
    "id": {
      "$ref": "db://custom.json#/id.json"
    },
    "foo": {
      "$ref": "/var/tmp/json-deref-schema-tests/foo.json"
    },
    "bar": {
      "$ref": "#/definitions/bar"
    },
    "rfc1808": {
      "$ref": "ftp://ftp.is.co.za/rfc/rfc1808.txt"
    }
  }
}