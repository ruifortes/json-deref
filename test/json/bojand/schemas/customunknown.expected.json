{
  "description": "Just a basic schema.",
  "title": "Basic Object",
  "type": "object",
  "definitions": {
    "bar": {
      "description": "bar property",
      "type": "boolean"
    }
  },
  "properties": {
    "id": {
      "description": "custom unique identifier of a the object",
      "type": "string",
      "minLength": 1,
      "readOnly": true
    },
    "foo": {
      "description": "foo property",
      "readOnly": true,
      "type": "number"
    },
    "bar": {
      "description": "bar property",
      "type": "boolean"
    },
    "rfc1808": {
      "$ref": "ftp://ftp.is.co.za/rfc/rfc1808.txt"
    }
  }
}